jerry-core/vm/vm-stack.c

```diff
@@ -310,21 +324,17 @@ vm_get_context_value_offsets (ecma_value_t *context_item_p) /**< any item of a c
     {
       return PARSER_TRY_CONTEXT_STACK_ALLOCATION;
     }
-#if ENABLED (JERRY_ESNEXT)
```

Comment:
Why do we need to remove these?

Reply:
The entire function is guarded by this, there is no need extra guards inside.